src/webhooks/components/WebhooksDetailsPage/WebhooksDetailsPage.tsx

```diff
@@ -62,58 +69,83 @@ const WebhooksDetailsPage: React.FC<WebhooksDetailsPageProps> = ({
 
   return (
     <Form initial={initialForm} onSubmit={onSubmit}>
-      {({ data, hasChanged, submit, change }) => (
-        <Container>
-          <Backlink onClick={onBack}>{appName}</Backlink>
-          <PageHeader
-            title={
-              isUnnamed(webhook)
-                ? intl.formatMessage({
-                    defaultMessage: "Unnamed Webhook Details",
-                    description: "header"
-                  })
-                : intl.formatMessage(
-                    {
-                      defaultMessage: "{webhookName} Details",
+      {({ data, hasChanged, submit, change }) => {
+        const syncEventsChoices = disabled
+          ? []
+          : mapSyncEventsToChoices(Object.values(WebhookEventTypeSync));
+        const asyncEventsChoices = disabled
+          ? []
+          : mapAsyncEventsToChoices(
+              Object.values(WebhookEventTypeAsync),
+              data.asyncEvents
+            );
+
+        const handleSyncEventsSelect = createSyncEventsSelectHandler(
+          change,
+          data.syncEvents,
+          syncEventsChoices
+        );
+        const handleAsyncEventsSelect = createAsyncEventsSelectHandler(
+          change,
+          data.asyncEvents,
+          asyncEventsChoices
+        );
```

Comment:
Logic here seems to be the same as in `WebhookCreatePage.tsx`. I would extract it to a separate hook

Reply:
Indeed, ...I went further and removed the whole `WebhookCreatePage`, it is the same as `WebhookDetailsPage`